src/cpu/aarch64/matmul/acl_lowp_matmul.hpp

```diff
@@ -0,0 +1,268 @@
+    arm_compute::Tensor bia_tensor;
+    arm_compute::Tensor dst_tensor;
+};
+
+struct acl_lowp_matmul_conf_t {
+    arm_compute::TensorInfo src_tensor_info;
+    arm_compute::TensorInfo wei_tensor_info;
+    bool with_bias;
+    arm_compute::TensorInfo bia_tensor_info;
+    arm_compute::TensorInfo dst_tensor_info;
+};
+
+status_t configure_gemm(
+        acl_lowp_matmul_obj_t &acl_obj, const acl_lowp_matmul_conf_t &almc) {
+
+    acl_obj.src_tensor.allocator()->init(almc.src_tensor_info);
+    acl_obj.wei_tensor.allocator()->init(almc.wei_tensor_info);
+    if (almc.with_bias) {
+        acl_obj.bia_tensor.allocator()->init(almc.bia_tensor_info);
+    }
+    acl_obj.dst_tensor.allocator()->init(almc.dst_tensor_info);
+
+    acl_obj.gemm.configure(&acl_obj.src_tensor, &acl_obj.wei_tensor,
+            almc.with_bias ? &acl_obj.bia_tensor : nullptr,
+            &acl_obj.dst_tensor);
+
+    return status::success;
+}
+
+struct acl_lowp_matmul_resource_t : public resource_t {
+    acl_lowp_matmul_resource_t()
+        : acl_obj_(utils::make_unique<acl_lowp_matmul_obj_t>()) {}
+
+    status_t configure(const acl_lowp_matmul_conf_t &almc) {
+
+        if (!acl_obj_) return status::out_of_memory;
+
+        acl_obj_->src_tensor.allocator()->init(almc.src_tensor_info);
+        acl_obj_->wei_tensor.allocator()->init(almc.wei_tensor_info);
+        if (almc.with_bias) {
+            acl_obj_->bia_tensor.allocator()->init(almc.bia_tensor_info);
+        }
+        acl_obj_->dst_tensor.allocator()->init(almc.dst_tensor_info);
+
+        acl_obj_->gemm.configure(&acl_obj_->src_tensor, &acl_obj_->wei_tensor,
+                almc.with_bias ? &acl_obj_->bia_tensor : nullptr,
+                &acl_obj_->dst_tensor);
+
+        return status::success;
+    }
+
+    acl_lowp_matmul_obj_t &get_acl_obj() const { return *acl_obj_; }
+
+    DNNL_DISALLOW_COPY_AND_ASSIGN(acl_lowp_matmul_resource_t);
+
+private:
+    std::unique_ptr<acl_lowp_matmul_obj_t> acl_obj_;
+};
+
+struct acl_lowp_matmul_t : public primitive_t {
+    struct pd_t : public dnnl::impl::cpu::matmul::cpu_matmul_pd_t {
+
+        pd_t(const matmul_desc_t *adesc, const primitive_attr_t *attr,
+                const cpu_matmul_pd_t *hint_fwd_pd)
+            : cpu_matmul_pd_t(adesc, attr, hint_fwd_pd), almc_() {}
+
+        using cpu_matmul_pd_t::cpu_matmul_pd_t;
+
+        DECLARE_COMMON_PD_T(
+                "lowp_gemm:acl", acl_lowp_matmul_t, USE_GLOBAL_SCRATCHPAD);
+
+        status_t init(engine_t *engine) {
+
+            VDISPATCH_MATMUL(
+                    set_default_formats(), "failed to set default formats");
+            using smask_t = primitive_attr_t::skip_mask_t;
+            VDISPATCH_MATMUL(attr()->has_default_values(smask_t::scales_runtime
+                                     | smask_t::zero_points_runtime),
+                    "only scale and zero point attrs supported");
+            VDISPATCH_MATMUL(attr()->scales_.get(DNNL_ARG_SRC).mask_ == 0
```

Comment:
```suggestion
            VDISPATCH_MATMUL(attr()->scales_.has_default_values({DNNL_ARG_SRC, DNNL_SRG_WEIGHTS})
                    && attr()->zero_points_.has_default_values({DNNL_ARG_SRC, DNNL_SRG_WEIGHTS})
                    && attr()->scales_.get(DNNL_ARG_SRC).mask_ == 0
```
It might require to apply clang-format on top of the suggestion.

The API: https://github.com/oneapi-src/oneDNN/blob/94272ffcd603207864143b476d780b0214532c76/src/common/primitive_attr.hpp#L262

Reply:
Resolving as there is a check below